Client/src/main/java/org/projectcardboard/client/controller/Client.java

```diff
@@ -75,7 +86,10 @@ public void onTextMessage(WebSocket websocket, String message) {
         this.ws.connect();
         break;
       } catch (WebSocketException e) {
-        System.out.println(e.getMessage());
+
+        logger.warn("Failed to Connect");
+        logger.trace(e.getMessage());
```

Comment:
I think pretty much everything being logged at `trace` should be at `debug`. Usually in my experience `trace` is reserved for when you want a bunch of output from your _dependencies_ as well as your application while `debug` is for when you want debug output from your application.